volatility/framework/symbols/windows/extensions/__init__.py

```diff
@@ -666,3 +700,81 @@ def to_list(self,
 
     def __iter__(self) -> Iterator[interfaces.objects.ObjectInterface]:
         return self.to_list(self.vol.parent.vol.type_name, self.vol.member_name)
+
+
+class TOKEN(objects.StructType):
+    """A class for process etoken object."""
+
+    def get_sids(self) -> Iterable[str]:
+        """Yield a sid for the current token object."""
+
+        userAndGroupCount = int(self.UserAndGroupCount)
```

Comment:
I don't think this needs casting as an int?  Is there a specific reason this was necessary?